assemblies/platform/pentaho-cdf-dd-samples/src/main/resources/plugin-samples/pentaho-cdf-dd/legacy/tests/GMapsOverlay/gMapsOverlay.cdfde

```diff
@@ -427,7 +427,7 @@
      },
      {
       "name": "mapName",
-      "value": "/public/plugin-samples/pentaho-cdf-dd/tests/GMapsOverlay/map-def/chicago_beats.js",
```

Comment:
Can't this be something like is done on the real time sample:
"value": "${solution:resources/js/categoryDetailsTooltip.js}"
??